ci/jobs/satellite6-upgrade-pulp.yaml

```diff
@@ -0,0 +1,194 @@
+# Macro that triggers jobs on Satellite Jenkins.
+#
+# Macro parameters:
+#
+#   job_name: the name of the job that should be triggered. For example:
+#
+#     job_name: satellite6-installer
+#
+#   job_parameters: the parameters that should be passed to the job if
+#     required. The format should be like below otherwise a Python exception
+#     could be raised:
+#
+#     job_parameters: >
+#         'parameter1': 'value1',
+#         'parameter2': 'value2',
+#         'parameter3': 'value3',
+#
+#     The quotes around parameters and values are required. This is a Python
+#     dictionary definition. The wrapping curly brackets is provided by the
+#     macro, because that, don't add them.
+#
+- builder:
+    name: satellite-jenkins-trigger
+    builders:
+        - shining-panda:
+            build-environment: virtualenv
+            name: satellite6-jenkins-trigger
+            python-version: System-CPython-2.7
+            nature: shell
+            command: |
+                pip install -U pip requests
+        - shining-panda:
+            build-environment: virtualenv
+            name: satellite6-jenkins-trigger
+            python-version: System-CPython-2.7
+            nature: python
+            command: |
```

Comment:
I was thinking that just about any time we saw `command: |` followed by a wall of text we'd switch to including from the scripts dir, so we should probably do that here, as well.